### PR TITLE
Create stars

### PR DESCRIPTION
**Pull request by ivan-kadykalo**
- [DEMO LINK](https://ivan-kadykalo.github.io/layout_stars/)
- [TEST REPORT LINK](https://ivan-kadykalo.github.io/layout_stars/report/html_report/)

### CODE REVIEW

**Comment by ivan-kadykalo** [comment]
I have tried to use gap: 4px in flex but it doesn't past a tests, and it's looks strange because in a viewport it looks the same as with margin-right. 
I'm doing something wrong or it is problem with a tests? 
The part of code with gap i left in the code, look on it please.

File: src/index.html [hunk 1]
```diff
@@ -8,6 +8,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <main class="container">
```
Comment (illiarerun) [inline src/index.html:11]: you don't really need this container, remove it

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
+* {
+  margin: 0;
+}
```
Comment (illiarerun) [inline src/style.css:3]: don't reset your margins with * selector, it's bad practice, you only need to reset margin for body in this task

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  box-sizing: border-box;
```
Comment (illiarerun) [inline src/style.css:8]: box-sizing should be the same for all elements, that's the property you can change with * selector

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center center;
+}
+
+.stars {
+  display: flex;
+  /* Don`t past a tests but looks correct */
+  /* gap: 4px; */
```
Comment (illiarerun) [inline src/style.css:17]: The thing with flex-gap is that not all of the browsers support it yet, that's why tests doesn't take it to account

**Review by illiarerun — APPROVED** [review]

**Pull request closed without merging**